### PR TITLE
VPN landing page headline experiment [fix #14131]

### LINKED ISSUE #14131: VPN LP - Heading/Subheading test
Hello. The PXI team would like to conduct a 3-way split on the following headings and sub-headings for the [VPN landing page:](https://www.mozilla.org/en-US/products/vpn/?entrypoint_experiment=vpn-pricing-position&entrypoint_variation=2)

**Option 1** (control/current variant)
_H1:_ Powerful privacy for peace of mind

**Option 2**
_H1:_ Powerful privacy for peace of mind `_(keep as is)_`
_H2:_ Switch on complete web activity and location protection from hackers and trackers.

**Option 3**
_H1:_ Mozilla VPN
_H2:_ Protect your digital activity and physical location with just one click.

There currently does not exist a subheading on the page. For **Option 2 **and** Option 3,** we will need to determine the best placing for the subheading. This is at the developer's discretion. 

There is no set timeline for this, but it would be nice to have it done when we have a bit of time to cross it off our list. 

Please let me know if you have any questions. Thank you!

### PR DESCRIPTION
**Pull request by craigcook**
## One-line summary
Adds a Traffic Cop experiment to test three different headline treatments on the VPN landing page (English only).
Also removes the previous pricing position experiment and moves the pricing section to the top of the page (which was the winning option).

## Issue / Bugzilla link
#14131
#14176

## Testing
v1 (control): http://localhost:8000/en-US/products/vpn/?entrypoint_experiment=vpn-headlines&entrypoint_variation=1
v2: http://localhost:8000/en-US/products/vpn/?entrypoint_experiment=vpn-headlines&entrypoint_variation=2
v3: http://localhost:8000/en-US/products/vpn/?entrypoint_experiment=vpn-headlines&entrypoint_variation=3

### CODE REVIEW

**Comment by craigcook** [comment]
Marking Do Not Merge. We're going to add French and German to the test, just need to get those strings back.

**Comment by craigcook** [comment]
Added German and French to the headline experiment, and also removed the previous pricing position experiment (moving pricing to the top, which was the clear winner). This is ready for review now.

File: bedrock/products/templates/products/vpn/landing-refresh.html [hunk 1]
```diff
@@ -83,7 +94,15 @@
     media_class='mzp-l-split-h-center',
     media_after=True
   ) %}
+  {% if entrypoint_experiment == 'vpn-headlines' and entrypoint_variation == '2' %}
+    <h1 class="u-title-xl">{{ ftl('vpn-landing-powerful-privacy-for-peace') }}</h1>
+    <h2 class="c-page-subtitle u-title-xs">{{ headline_exp_v2_sub }}</h2>
+  {% elif entrypoint_experiment == 'vpn-headlines' and entrypoint_variation == '3' %}
+    <h1>Mozilla VPN</h1>
+    <h2 class="c-page-subtitle u-title-xs">{{ headline_exp_v3_sub }}</h2>
```
Comment (reemhamz) [inline bedrock/products/templates/products/vpn/landing-refresh.html:102]: **Suggestion(design/non-blocking):** I think based on how we have most of our other pages laid out, we don't usually have our subheadings bolded (granted, they're usually `p` tags), so I'm thinking keeping the font-weight normal here would be more consistent with some of our other pages:

<kbd>Screenshot 2024-02-07 at 11 57 31 AM</kbd>

This is up to you to implement as it's at your discretion, but thought it'd be a good suggestion to throw in 👍🏼 plus I think it's a nice separator from the page title

**Review by reemhamz — APPROVED** [review]
Honestly, you did a pretty great job implementing this change 🎉 

I have a minor, non-blocking suggestion for you, but other than that it's a r+(wc?)

File: bedrock/products/templates/products/vpn/landing-refresh.html (hunk 1 above)
Comment (craigcook) [reply bedrock/products/templates/products/vpn/landing-refresh.html:102]: Good suggestion. Done.

**Pull request merged**